os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -364,6 +364,10 @@ static void up_dumpstate(void)
 
 	if (sp > ustackbase || sp <= ustackbase - ustacksize) {
 		lldbg("ERROR: Stack pointer is not within the allocated stack\n");
+		up_stackdump(ustackbase - ustacksize + 1, ustackbase);
```

Comment:
How about adding the log "Original? Proper Stack Dump" to say what they are clearly?

Reply:
ok